gaas-ui/src/components/view-graphs/view-graphs-table.tsx

```diff
@@ -146,8 +154,8 @@ function MainGraphTableRow(props: IGraphRow) {
                     }}> {graph.getConfigName().charAt(0).toUpperCase()}</Avatar></TableCell>
                 <TableCell aria-label={"graph-status"}>
                     <StatusChip status={graph.getStatus()}/></TableCell>
-                <TableCell aria-label={"graph-url"}><a href={graph.getUrl()} target="_blank"
-                                                       rel="noreferrer">{graph.getUrl()}</a></TableCell>
+                <TableCell aria-label={"graph-url"}>
+                    <a id={graph.getId()} target="_blank" rel="noopener noreferrer">{graph.getUrl()}</a></TableCell>
```

Comment:
Still getting es-lint warnings

Reply:
Added a placeholder href to fix the issue.